bpmn-model/src/main/java/io/zeebe/model/bpmn/util/time/TimeDate.java

```diff
@@ -23,9 +27,17 @@ public TimeDate(Interval interval) {
     this.interval = interval;
   }
 
+  public static Timer parse(String timeDate) {
```

Comment:
Return type here is unexpected, considering we're in the context of `TimeDate`, I would return a `TimeDate` here, which is anyway a `Timer` implicitly.